_posts/2019/2019-08-29-why-hosting-townhall-meetings-is-a-good-idea.md

```diff
@@ -0,0 +1,58 @@
+date: 2019-08-29
+header_image: private/townhall.jpg
+header_position: center
+header_overlay: true
+category: on-the-job
+tags: ["company", "communication", "meeting"]
+authors: ["Birgit"]
+about_authors: ["bbader"]
+---
+
+Back then when I was new to ePages, I got to know a new kind of meeting that took place at the end of every week.
+The "Tea Time", as it was called at that time, was held every Friday at 4:00 p.m. in our big conference room.
+Actually, it was one of the very first things that I was told about, when I started my job here.
+It was THE place to personally introduce myself to the company.<br>
+But of course, introducing new colleagues to the company was not the only thing we did there.
+The various teams were assigned presentation slots throughout the year to speak about a topic that was relevant to their work.
+So one day we were told about the latest news from Marketing, while another Friday the Design Services team showed online shops of our merchants, or Product Management presented the new roadmap.
+
+## Up-to-dateness
+
+Why am I telling you that?
+Because I find this meeting a great opportunity for regular company updates and communication amongst employees.
+Nowadays we call it "Town Hall", and we shifted the schedule to 1:00 p.m. so that colleagues from all locations are able to join.
+As a company with various development sites we have the challenge to meet everyone's schedule, and the time around noon turned out to be the best fit for the highest attendance rate.
+And a high employee attendance makes total sense if you want to bring information across, right? 😉
+
+## Socializing
+
+At the end of a Town Hall meeting there's always time for a chat or an exchange with other colleagues while having a "weekend-is-coming-soon" drink.
+Or we just go for a quick table soccer or table tennis session before heading back to our work places.
+
+## Visible managers
+
+Having all employees in one room is also a good opportunity for the management to address certain important topics, or to answer questions that are on the employees' minds.
+Although we have flat hierarchies and our managers' doors are always open, these meetings are perfect for being visible to staff and available for questions and feedback.
+
+There are even dedicated CEO slots, where Wilfried talks about his product vision, important partner meetings, relevant exhibitions coming up, or even how to shape the future of ecommerce.
+
+## Caring for culture and values
+
+Every once in a while the regular Town Hall meeting is replaced by either a company breakfast or an International Town Hall.
+Breakfast takes place every two months 9:30 a.m. on fridays.
+Our office manager lovingly arranges bread rolls, cold cuts, müsli, juices, fruit, and vegetables, and we all have the chance to catch up with other team members, and enjoying a great start into the last day of the work week.
+
+Our International Town Hall takes place every two months 5:00 p.m. on thursdays.
```

Comment:
Thursdays

Would even add an "at" before 5:00 p.m. to make it easier to "digest" all this data.